feature/experimental/policy/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,311 @@
+  * No high CPU spike or usage on control or forwarding plane
+  * No high memory utilization or usage on control or forwarding plane
+  * No processes/daemons high CPU/Memory utilization
+  * No generic drop counters
+    * QUEUE drops
+    * Interfaces
+    * VOQ
+    * Fabric drops
+    * ASIC drops
+  * No flow control frames tx/rx
+  * No CRC or Layer 1 errors on interfaces
+  * No config commit errors
+  * No system level alarms
+  * In spec hardware should be in proper state
+    * No hardware errors
+    * Major Alarms
+  * No HW component or SW processes crash
+Openconfig-paths for the above are covered in the `OpenConfig Path and RPC Coverage` section below<br><br><br>
+
+
+
+
+
+**RT-3.33.1: [Baseline] Traffic flow between ATE1:Port1 and ATE2:Port2 via DUT's Default VRF**
+  * Please run all the health checks suggested above prior to starting the test as a baseline.
+  * Start all the Flows from ATE1:Port1 to DUT:Port1 
+  * DUT has the following VRF selection policy
+    * Statement1: traffic matching any IPV4 0r IPV6 destination with any DSCP value, Punt to default vrf <br>
+  * Expectations:
+    * All traffic received in the Default VRF destined for any IPv4Prefix is sent back to ATE2:Port2 based on LPM lookup of EBGP learnt routes. 
+    * Run post test Health checks above to ensure no anomalies<br><br><br>
+
+
+**RT-3.33.2: Identify traffic by DSCP (BE1) and punt to non-default VRF**
+  * Please run all the health checks suggested above prior to starting the test.
+  * Start all the Flows from ATE1:Port1 to DUT:Port1 
+  * DUT has the following VRF selection policy
+    * Statement1: Traffic matching any IPV4 or IPV6 destination with DSCP value BE1, Punt to non-default vrf
+  * Statement2: Traffic matching any IPV4 or IPV6 destination with DSCP value AF1,AF2,AF3,AF4, Punt to default vrf <br>
+  * Expectations:
+    * All traffic received in the Default VRF destined for any Destination but with DSCP value BE1 is punted to Non-Default VRF and sent back to ATE2:Port3 based on LPM lookup of EBGP learnt routes. 
+ * All traffic received in the Default VRF destined for any Destination but with DSCP value AF1,AF2,AF3,AF4 is sent back to ATE2:Port2 based on LPM lookup of EBGP learnt routes. 
+
+    * Run post test Health checks above to ensure no anomalies<br><br><br>
+
+
+**RT-3.33.3: Identify traffic by DSCP (BE1,AF1,AF2,AF3,AF4) and punt to non-default VRF**
+
+
+* Please run all the health checks suggested above prior to starting the test.
+  * Start all the Flows from ATE1:Port1 to DUT:Port1 
+  * DUT has the following VRF selection policy
+    * Statement1: Traffic matching any IPV4 or IPV6 destination with DSCP value BE1,AF1,AF2,AF3,AF4 Punt to non-default vrf<br>
+  * Expectations:
+    * All traffic received in the Default VRF destined for any Destination but with DSCP value BE1,AF1,AF2,AF3,AF4 is punted to Non-Default VRF and sent back to ATE2:Port3 based on LPM lookup of EBGP learnt routes. 
```

Comment:
It's not clear to me (see longer comment above) how this test setup actually results in this happening. What is causing the non-default VRF to select ATE2:Port3 for the same flows?

Reply:
Had to change the complete README, appreciate if you can take a look on the latest updated README.